ext/com_dotnet/php_com_dotnet_internal.h

```diff
@@ -87,7 +87,7 @@ PHP_COM_DOTNET_API OLECHAR *php_com_string_to_olestring(char *string,
 
 
 /* com_com.c */
-PHP_FUNCTION(com_create_instance);
+const zend_function_entry php_com_com_funcs[];
```

Comment:
I'd suggest to move the definition of these arrays directly into `com_extension.c` instead. There doesn't seem to be much reason to have them in other files.

Reply:
Makes perfect sense. Thanks!